exercises/practice/anagram/test/anagram_test.dart

```diff
@@ -1,31 +1,21 @@
-import 'package:anagram/anagram.dart';
 import 'package:test/test.dart';
+import 'package:anagram/anagram.dart';
```

Comment:
The imports should be sorted alphabetically

Reply:
I noticed that too, but I purposefully did not mention it. I thought we should add a [lint rule](https://dart.dev/tools/linter-rules#directives_ordering) for that so `dart format --fix` takes care of it.

Reply:
Since this is a generated file, it's not trivial to know the ordering of these two imports and effectively sort them in a template. I think the template would need to add the aforementioned lint rule to `analysis_options.yaml` of the generated exercise and apply `dart format --fix`

Reply:
That is easy enough to do. Is there an example of where this is done correctly so that `dart format --fix` will work?

Reply:
I tried adding `directives_ordering` to the linter rules in the `analysis_options.yaml` file that is in the `exercises/practice/anagram` directory, like so:

```
linter:
  rules:
    # Error Rules
    - avoid_relative_lib_imports
    - avoid_types_as_parameter_names
    - literal_only_boolean_expressions
    - no_adjacent_strings_in_list
    - valid_regexps
    
    # Style rules
    - directives_ordering
 ```

Then I ran `dart format --fix exercises/practice/anagram`, but it didn't change the ordering.

```
$ dart format --fix exercises/practice/anagram/
Formatted 2 files (0 changed) in 0.18 seconds.
```

Any suggestioons?

Reply:
The lint rule `directives_ordering` has a quick fix instead of a style fix. Quick fixes are applied by `dart fix --apply`. Style fixes are applied by `dart format -l 120 .`. You can think of `dart format` as [prettier](https://prettier.io/) version of dart and `dart fix` as the [ESLint](https://eslint.org/) version of dart.

Reply:
Thank you, that worked! I'll push this up again.